samples/java-customer-registry/README.md

```diff
@@ -11,7 +11,7 @@ To run the example locally:
     sbt sdk/publishM2
     export AKKASERVERLESS_JAVA_SDK_VERSION="0.7.0-beta....-SNAPSHOT"
     cd samples/java-customer-registry
-    mvn compile exec:java
+    mvn compile exec:java  -DmainClass="customer.Main"
```

Comment:
Would it work with a `<properties><exec.mainClass>customer.Main</exec.mainClass></properties>` in the pom.xml instead (and remove the reference to ${mainClass} in the exec plugin config?